### ISSUE TITLE
HTMLparser module

### ISSUE THREAD

**Issue by Luka-Filipovic** [ungoogled-software/ungoogled-chromium-windows#18]
```
ModuleNotFoundError: No module named 'HTMLParser'
[152/21839] CXX obj/base/base/base_jumbo_5.obj
ninja: build stopped: subcommand failed.

```
Not sure why am I getting this issue. HTMLParser is a python 2 module, but I guess it should work with python 3 with 2to3.
Did anyone have a same issue?

**Comment by tangalbert919** [comment]
Sounds like a problem with `tools/metrics/histograms/extract_histograms.py`. Unfortunately, that's not the only Python 2 script that needs to be fixed so it runs on Python 3.

Right now, I have a patch in my fork (on the `python3-support` branch) that can fix those scripts, but it's incomplete. Progress on Python 3 migration on the [Chromium Bug Tracker](https://bugs.chromium.org/p/chromium/issues/list?q=Proj%3DPython3Migration%20&can=2) is currently a bit slow-going.

**Comment by Luka-Filipovic** [comment]
Thanks for a reply! 
Yes, the histogram file was causing a problem. 

Since it's a known issue, how do you ever build it with python3?

**Comment by Luka-Filipovic** [comment]
It works by setting python3 as `py `and python2.7 as `python `in `path`.
[Here ](https://cloud.mail.ru/public/BKPK/53joir6BY)is the latest version to download.

**Comment by tangalbert919** [comment]
Don't worry. I still use the old build instructions to build, since that's the only way to get this browser built at the moment.

**Comment by Eloston** [comment]
@tangalbert919 If that's the case, we should update the README. Otherwise, it'll be frustrating for users who are trying to build.

**Comment by NikolaF94** [comment]
> Don't worry. I still use the old build instructions to build, since that's the only way to get this browser built at the moment.
Can you share that old build instructions and old build.py?

**Comment by Eloston** [comment]
Fixed in #19
